cmds/bmp/bms_command.go

```diff
@@ -115,7 +114,7 @@ func (cmd bmsCommand) Execute(args []string) (int, error) {
 	for _, value := range content {
 		table.Append(value)
 	}
-	cmd.ui.PrintTable(table)
+	table.Render()
```

Comment:
I don't really like this. Reason is that now the table has to be created with `os.Stdout` and I want to have all output controlled by `cmd.Ui` interface. One thing you could do is to have the `Ui` interface return a new `TableWriter` and use there the `os.Stdout`. So like 98 becomes:

```
table := cmd.ui.NewTableWriter()
```